IFC4x3/Properties/w/Width_0HGpSpErnC4QCvVLT__1Q3/DocProperty.xml

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Width" UniqueId="11433733-3b5c-4c11-a339-7d577ef81683" PrimaryDataType="IfcPositiveLengthMeasure" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Width_0HGpSpErnC4QCvVLT__1Q3" Name="Width" UniqueId="11433733-3b5c-4c11-a339-7d577ef81683" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
Is Width an actual measure in addition to the NominalWidth? Isn't it then a quantity?